app/controllers/agent/CheckYourAnswersController.scala

```diff
@@ -53,9 +52,9 @@ class CheckYourAnswersController @Inject()(val authService: AuthService, subscri
         }
     }
 
-  def backUrl(incomeSource: Option[IncomeSourceType])(implicit request: Request[_]): String = {
+  def backUrl(incomeSource: Option[IncomeSourceModel])(implicit request: Request[_]): String = {
     incomeSource match {
-      case Some(Business) =>
+      case Some(IncomeSourceModel(true, false, false)) =>
```

Comment:
This is more of a generic ask across every case you have this, could you replace
```suggestion
      case Some(IncomeSourceModel(true, false, _)) =>
```
On all cases where you do not care about the value of foreign property, which for now should be all cases

Reply:
done